consensus-types/types/payload_header_test.go

```diff
@@ -245,7 +245,7 @@ func TestExecutionPayloadHeader_NewFromSSZ_Invalid(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			buf := tc.malleate()
 			_, err := (&types.ExecutionPayloadHeader{}).NewFromSSZ(buf, version.Deneb())
-			require.ErrorContains(t, err, tc.expErr.Error())
```

Comment:
brittle tests, I enriched an error description and they failed. `ErrorIs` is the idiomatic way to do this AFAIK